### ISSUE TITLE
Android Samples are not allowing the debugger to attach.  

### ISSUE THREAD

**Issue by intelzombi** [LunarG/VulkanSamples#137]
I filed a bug with the Google team on this issue because I originally felt it was an Android Studio issue most likely.  However they did root cause the issue and it is caused by the length of the application names. The details of the issue are located in the google issue tracker:
https://issuetracker.google.com/issues/62035415

Thought you should know.  I want to see your samples become great.
Thanks.

**Comment by ggfan** [comment]
I will get this one fixed

**Comment by ggfan** [comment]
This should now be fixed with https://github.com/LunarG/VulkanSamples/pull/140

please kindly check and help to close the issue. thanks

**Comment by intelzombi** [comment]
Great Job gg.  I did an rm -rf on the entire sample set to make sure I didn't have any lingering stuff that would give any kind of false positive or something.  I did a git clone on the sample set.  I was able to compile without error and I set a debug break point in the native side and it hit it without issue so from my perspective everything looks great.

Thanks so much for fixing these issues so quickly
iz :)
